File: jar-client/src/main/java/com/beancounter/client/ingest/RowAdapter.java
package com.beancounter.client.ingest;

import com.beancounter.common.input.TrnInput;
import com.beancounter.common.input.TrustedTrnImportRequest;

public interface RowAdapter {
  TrnInput transform(TrustedTrnImportRequest trustedTrnImportRequest);

}

File: jar-client/src/main/java/com/beancounter/client/ingest/RowAdapter.kt
package com.beancounter.client.ingest

import com.beancounter.common.input.TrnInput
import com.beancounter.common.input.TrustedTrnImportRequest

interface RowAdapter {
    fun transform(trustedTrnImportRequest: TrustedTrnImportRequest?): TrnInput?
}
